Methane_MarkDown.Rmd

```diff
@@ -33,9 +34,7 @@ After creating and running the core we query it for methane-related outputs
 between the years of 1745 and 2100. Hector has several options that relate to methane, including emissions, concentration, postindustrial, natural, soil lifetime and stratosphere lifetime.
 
 ```{r Emissions-data-frame}
-results<- fetchvars(core, 1745:2100, c(EMISSIONS_CH4(),
-                                       CONCENTRATIONS_CH4()))
-invisible(shutdown(core))
+results<- fetchvars(core, 1745:2100, c(EMISSIONS_CH4(), CONCENTRATIONS_CH4()))
 head(results)
```

Comment:
Maybe add a sentence in your text explaining what this output is? (Rmd comment)

Reply:
+1